student.rb

```diff
@@ -1,5 +1,4 @@
-require_relative 'person'
-
+require_relative 'Person'
```

Comment:
- [ ] Nice work creating the app however the `Person` should be `person` here as the name of the file as for now the app produces an error 👇🏼, but after changing the name the app works as expected. kindly consider making the change so your app works properly

![Screenshot_20230603_150830](https://github.com/LoviMadaan/oop-school-library/assets/25227022/78504c75-e2a1-4625-b36a-bf9a85f6690c)

Reply:
Thank you so much.. I have implemented the change.